lib/ain-evm/src/storage/block_store.rs

```diff
@@ -257,25 +257,28 @@ impl BlockStore {
         }
     }
 
-    fn dump_all(&self, limit: usize) {
+    fn dump_all(&self, limit: usize) -> String {
+        let mut out = String::new();
         for arg in &[
             DumpArg::Blocks,
             DumpArg::Txs,
             DumpArg::Receipts,
             DumpArg::BlockMap,
             DumpArg::Logs,
         ] {
-            self.dump(arg, None, limit);
+            out.push_str(self.dump(arg, None, limit).as_str());
```

Comment:
We're constructing a massive string here. This needs to be chunked. Better to use a buffered writer that just keeps writing out

Reply:
Switched to using fmt write to write to the string stream instead. To do in the future will be to shift out and use a streaming output on our RPC server.